readme.js

```diff
@@ -0,0 +1,77 @@
+const fs = require('fs');
+const ruleMod = require('./rules');
+
+function formatUsage(usage) {
+  return usage.replace(/~~~/g, '```').replace(/(^[ \t]*\n)/gm, '\n');
+}
+
+function createRuleList() {
```

Comment:
This function will go away if you approve the contents of `rules.js`. This function is here to show that the file has captured all the current rules since it managed to update the README file with only a few needed changes.